Dockerfile

```diff
@@ -108,7 +108,7 @@ ENV CMAKE_ARGS="-DCUDA_ARCH:STRING=${CUDA_ARCH}"
 
 COPY . /code/timemachine/
 WORKDIR /code/timemachine/
-RUN pip install --no-cache-dir -e .[test] && rm -rf ./build
+RUN pip install --no-cache-dir -e . && rm -rf ./build
```

Comment:
Resolved offline: `timemachine_dev` includes CUDA development dependencies needed to build the C++ extension module (but is not intended to include "developer tools" like `pytest`)

Reply:
Added comment in https://github.com/proteneer/timemachine/pull/1306/commits/e8cf7e2a4b9dd10da649d7c0d3603e7757fac8f9